### PR TITLE
Updated Readme variables and user creation data

### PR DESCRIPTION
**Pull request by rbillings**
Addresses issue #784

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ from the the project root, issue the following command:
 
     pip install -Ur requirements.txt
 
+###Create test users
+Some of the tests require logging in as an user, and others require logging in as an editor user or paypal user.
+To run these tests you will need to create accounts on https://addons.allizom.org/. If you want to run the tests
+that require editor access, then you will need to ask someone on irc to upgrade one of your test accounts.
```
Comment (stephendonner) [inline README.md]: Uber-nit: s/irc/IRC/

File: README.md [hunk 2]
```diff
@@ -69,6 +69,11 @@ from the the project root, issue the following command:
 
     pip install -Ur requirements.txt
 
+###Create test users
+Some of the tests require logging in as an user, and others require logging in as an editor user or paypal user.
+To run these tests you will need to create accounts on https://addons.allizom.org/. If you want to run the tests
+that require editor access, then you will need to ask someone on IRC to upgrade one of your test accounts.
```
Comment (davehunt) [inline README.md]: There's no need for an 'editor' user. The 'editable' user is a user reserved for editing the user profile without impacting tests that rely on the profile to be static. We're actually working on switching to Firefox Accounts for AMO, which will mean there's going to be no need for `variables.json` to include an AMO user. That said, we can still take this improvement to the README in the short term, but both 'default' and 'editable' are basic accounts.

File: README.md [hunk 3]
```diff
@@ -76,9 +81,31 @@ called `variables.json`, which we then pass to the tests via the command line.
...
 the `variables.json` file, which you will update to contain your own credentials.
 To do that, make a copy of the `variables.json` file which exists in the project root
- and update that with your own credentials. 
+ and update that with your own credentials (see above). 
  You will then pass the name of that file on the command line. 
  For the purposes of the examples below, assume you named your copy of the file `my_variables.json`.
+ 
+ The following is an example JSON file with the values missing. You can use this as a template.
+ 
+ {
```
Comment (davehunt) [inline README.md]: We should format this as JSON, using:

    ```json
    {
      ...
    }
    ```

File: README.md [hunk 4]
```diff
@@ -69,6 +69,11 @@ from the the project root, issue the following command:
 
     pip install -Ur requirements.txt
 
+###Create test users
```
Comment (davehunt) [inline README.md]: Nit: Space between # and heading.

File: README.md (hunk 2 above)
Comment (davehunt) [inline README.md]: We might want to add a note about how to create a Paypal user, though I don't really know how we do that. I suspect @krupa can help here.

**Comment by rbillings** [comment]
@mozilla/web-qa-sorcerers r? I'll fix the merge conflicts after a review.

**Comment by davehunt** [comment]
Sorry @rbillings, I rewrote this section when we changed the suite to use the API to create users for the tests - we no longer need to use the variables file for user credentials. It's only used for API credentials and PayPal credentials now. I'm going to close this as the changes no longer reflect the state of the test suite.

**Pull request closed without merging**